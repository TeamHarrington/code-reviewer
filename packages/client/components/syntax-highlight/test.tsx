import { SyntaxHighlight } from '.'
import renderer from 'react-test-renderer'
import React from 'react'

test('Syntax Highlight Component', () => {
  const component = renderer.create(<SyntaxHighlight codeString="" />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
