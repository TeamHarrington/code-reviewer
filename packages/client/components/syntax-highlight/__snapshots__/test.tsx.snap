// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Syntax Highlight Component 1`] = `
<div
  className="MuiGrid-root syntax-highlight__Container-sc-12l7p11-0 bJhvaZ MuiGrid-container"
>
  <div
    className="MuiGrid-root syntax-highlight__LeftContainer-sc-12l7p11-1 crXnUV"
  >
    <div
      className="MuiGrid-root annotation-indicators__Container-sc-16ay4gl-0 fSZeWP"
    >
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <div
        className="annotation-indicators__Row-sc-16ay4gl-1 kpAklh"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
    </div>
  </div>
  <div
    className="MuiGrid-root syntax-highlight__CodeContainer-sc-12l7p11-2 PEnnx"
  >
    <pre
      style={
        Object {
          "background": "#1E1E1E",
          "color": "#DCDCDC",
          "display": "block",
          "fontSize": "20px",
          "margin": 0,
          "overflowX": "auto",
          "padding": "0.5em",
        }
      }
    >
      <code
        style={
          Object {
            "float": "left",
            "paddingRight": "10px",
          }
        }
      >
        <span
          className="react-syntax-highlighter-line-number"
        >
          1

        </span>
      </code>
      <code />
    </pre>
  </div>
</div>
`;
